packages/remix-css-modules/README.md

```diff
@@ -0,0 +1,13 @@
+# Welcome to Remix!
```

Comment:
prolly one of the last tasks on the list but some documentation on how to use it might already be helpful for people like me wanting to test it out, this is what I figured out was the way to use it

```javascript
// in root.tsx
import CSSModules from "@remix-run/css-modules"

export const links: LinksFunction = () => {
  return [
    { rel: "stylesheet", href: CSSModules }
  ];
};
```

But I didn't manage to get the watch mode working, after a change in a css module I got a css-module-faker error. prolly WIP? 

big thanks for the work on this! we're very interested in using css modules in our remix app as well so if you need some help, let me know